Project-2/README.md

```diff
@@ -205,19 +205,25 @@ So Animals is bisimilar to Plants.
  
   (a) ∃r.C and ∀r.C
   
-  Answer: ∃r.C means that there exists a r-filler which is an element that belongs to C, while ∀r.C means that all r-fillers are Cs.
+  Answer: ∃r.C means that all x has a r-filler y which belongs to C, while ∀r.C means that all r-fillers ys of all x belong to C.
   
   (b) ∃r-.C and ∀r-.C
   
-  Answer: ∃r-.C means that there exists a inverse r-filler which an element that belongs to C, while ∀r.C means that all inverse r-fillers are Cs.
+  Answer: ∃r-.C means that there is a thing y r-related to all xs, and this thing belongs to C,
+  
+  while ∀r.C means that if there is a thing r-related to all xs, then this thing belongs to C.
   
   (c) <=nr and <=nr.C
   
-  Answer: <=nr means that there are no more than n r-fillers, while <=nr.C means that there are no more than n r-fillers in C.
+  Answer: <=nr means that role r connects all the xs to no more than n elements.
+ 
+  while <=nr.C means that role r connects all the xs to no more than n elements, and they belong to C.
   
   (d) ∃r-.C and ∃r-.{a} 
   
-  Answer: ∃r-.C means that there are no more than n inverse r-fillers, while <=nr.C means that there are no more than n inverse r-fillers in C.
+  Answer: ∃r-.C means that for all xs, there is at least a thing y, which is r-inverse related to it, and which belongs to C,
+  
+  while ∃r-.{a} means that for all xs, there is element a, which is r-inverse related to it.
```

Comment:
Hi Jieming, 

Question 2. As for your domain set Δ, it is better to say, Δ={ a, b, c, f, m, w }, and then treat “Aquatic animals = a, Seagull = b, Cats = c, Fish = f, Mammal = m, Whales = w, Fly = {b}, }” as your explanation of individual names.

Your T-box looks weird. You should not use two consecutive “≡” in “PP ≡ ≡ P ⊓ ¬P¯”, and you can delete a redundant “iPP ≡” in “iPP ≡ iPP ≡ (P ⊓ ¬P¯)¯”. Also, please add two “,” to differ among “iP ≡ P¯ O ≡ ∃P¯.(∃P) D ≡ ¬O”.

As for your A-box, you should change “pp(w, m)” to “PP(w, m)”, and change “pp(f, a)” to “PP(f, a)”. Also, delete “Fly: b” in you’re A-box because “a” is not a concept but an individual name. 

You don’t finish your model designing. What set of ordered pairs do “P”, “PP”, “iP”, “iPP”, “D” and “O” correspond to, respectively? You should say these.

Q3. (a) My new attempt: ∃R.(∀R.(∀R¯))

(b) My new attempt: ∃R¯.(∃R.(∃R¯)). Also, please delete “∃r.(∃r.C ∧ ∀r¯.C)” because there is no “C” in the formula (b).

(c) There is no “C” in the original formula so “∃r.(∃r.C ∧ ∀r¯.C)” must be wrong. Maybe you can say, ∀R.∃R.∀R.A.

(d) (∀R.A) ⊓ (∃R.B). This is very straightforward, please read Baader’s book, pp. 45 for further explanations.

Q4. ρ = {(V, INV), (WB, PO), (CB, CN)} is incomplete, you should add in it: (WB, AR), (CB, AN), (CB, MO). It is because a bisimulation is a mapping from all elements in a model to all elements in another model. So every element in bisimilar models must be mentioned. 

Here we wanna use a formula to differ between two models, so you can delete “≥2 ∃t.⊤” and keep “≥5 ∃t2.⊤”. Only one formula is enough because V in I1 does not satisfy “≥5 ∃t2.⊤” but INV in I2 satisfies it.

Q6. Note that in DL all quantified formulas include an implicit universal quantifier, so each quantified formula in DL must pick out “all individuals (xs) satisfying some complex condition”.

Q7. Interesting elucidation of Color-card. But you should tell us in what cases finite model property is important or unimportant. Finite model property (FMP) can be used to design a decidable algorithm for the satisfiability of its concepts.

Q8. How interesting your game is. Also, don’t forget to mention in what cases tree model property is important or unimportant. TMP is crucial to tableau-based algorithms for the consistency check of knowledge base and applied ontology. Most of applied ontologies, especially those BFO-based ones, have TMP, allowing that their consistency can be checked by tableau reasonings.

Q9-10. You can look at our Google Doc if you have some trouble. Good luck!

Reply:
Hi Karl,
Thank you for your comments!
Q2. Thanks for your pointing out my faults. In addition, did you mean that I should add sets of order pairs to, say PP, in T-box? such as ∃PP. Fly ⊑ T. And is it right that in order to complete these sets of order pairs, I should design some other concepts?
Q3. Your comment is really helpful!
Q4. Thank you for pointing out the complete mapping point!
Q7, Q8. Thanks for your noting that. - I didn't notice it,